### DISCUSSION TITLE
Leaderboard combo partially clips through accuracy when at high combo

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#20536] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/74088315/193158802-e12a0102-b1a7-4b38-8c25-e4e816ca68ad.png)
![image](https://user-images.githubusercontent.com/74088315/193158641-508ebd80-f43c-4be2-afc4-05c259ce906a.png)

This was mentioned in a previous discussion https://github.com/ppy/osu/discussions/20435#discussioncomment-3719492, so I thought it would make sense to open up a separate discussion to address that issue. I think that the accuracy and combo values should probably be a bit more spaced apart so that they don't end up clipping through each other, since it can look kind of weird when they're extremely close together.

**Comment by peppy** [comment, marked as answer]
Will be fixed by #20564.
